### PR TITLE
Client methods to get and update schema for an engine

### LINKED ISSUE #22: Client should be able to get and create/update engine schema
The python client should have the ability to get the current schema of an engine, or create/update new schema fields on an engine using the follwing app search endpoints.

`GET /api/as/v1/engines/{ENGINE_NAME}/schema`
`POST /api/as/v1/engines/{ENGINE_NAME}/schema`

### PR DESCRIPTION
**Pull request by CorrosiveKid**
Closes #22

### CODE REVIEW

**Comment by JasonStoltz** [comment]
Hey @CorrosiveKid, thank you for the contribution. Before we can merge this, we require that you sign a CLA: https://www.elastic.co/contributor-agreement.

**Comment by CorrosiveKid** [comment]
> Hey @CorrosiveKid, thank you for the contribution. Before we can merge this, we require that you sign a CLA: https://www.elastic.co/contributor-agreement.

Done!

File: swiftype_app_search/client.py [hunk 1]
```diff
@@ -107,6 +107,28 @@ def destroy_documents(self, engine_name, document_ids):
...
+
+        :param engine_name: Name of engine.
+        :return: Schema.
+        """
+        endpoint = "engines/{}/schema".format(engine_name)
+        return self.swiftype_session.request('get', endpoint)
+
+    def update_schema(self, engine_name, schema):
+        """
+        Create new schema fields or update the fields if they alreadt exists.
```
Comment (JasonStoltz) [inline swiftype_app_search/client.py]:
```suggestion
        Create new schema fields or update the fields if they already exists.
```

**Review by JasonStoltz — CHANGES_REQUESTED** [review]
I just noticed one small typo, it works great otherwise.

**Comment by CorrosiveKid** [comment]
> I just noticed one small typo, it works great otherwise.

Thanks for that, fixed!

**Review by JasonStoltz — APPROVED** [review]

**Pull request merged**